src/views/template/factory/factory.html

```diff
@@ -28,8 +28,17 @@
         </form>
     </div>
     <div class="panel-footer">
-        <div class="btn-group" role="group">
-            <button id="buy-truck" type="button" class="btn btn-success pull-right margin-top-10">Buy truck</button>
+        <div class="btn-group btn-group-justified" role="group">
+            <div class="btn-group" role="group">
+                <button id="size-truck" type="button" class="size-truck btn btn-default">
+                    <span class="text-success">{{#l}}Increase size trucks{{/l}}</span>
+                </button>
+            </div>
+            <div class="btn-group" role="group">
+                <button id="buy-truck" type="button" class="buy-truck btn btn-default">
```

Comment:
`id` is sufficient, so `buy-truck` can be removed from the class attributes. Same goes for the `size-truck` above.